common/types/activation.go

```diff
@@ -188,16 +174,30 @@ func NewActivationTx(
 	return atx
 }
 
-// SetMetadata sets ATXMetadata.
-func (atx *ActivationTx) SetMetadata() {
-	atx.Target = atx.TargetEpoch()
-	atx.MsgHash = BytesToHash(atx.HashInnerBytes())
+// ATXSigMsg is the data of ActivationTx that is signed.
+type ATXSigMsg struct {
```

Comment:
as discussed, lets leave ATXMetadata as is.
- use the original name (decided as common term for malfeasance in research)
- use Hash32 instead of ATXID to decouple ID calculation and signed data.

Reply:
> decided as common term for malfeasance in research

can we please stop doing this "decided in research" thing. if it was decided please specify it somewhere, and lets figure out if it really the best option